### ISSUE TITLE
Install framework using carthage

### ISSUE THREAD

**Issue by fedecastelli** [evermeer/EVReflection#264]
Hi,

when I'm trying to update frameworks using "Carthage update --platform iOS", I'm getting these errors:

```
Carthage/Checkouts/EVReflection/Source/EVReflection.swift:82:50: error: use of undeclared type 'EVCustomReflectable'
                    if var custom = original as? EVCustomReflectable {
                                                 ^~~~~~~~~~~~~~~~~~~
Carthage/Checkouts/EVReflection/Source/EVReflection.swift:334:34: error: use of undeclared type 'EVCustomReflectable'
        if let v = theObject as? EVCustomReflectable {
                                 ^~~~~~~~~~~~~~~~~~~
Carthage/Checkouts/EVReflection/Source/EVReflection.swift:866:24: error: use of undeclared type 'EVCustomReflectable'
        if theValue is EVCustomReflectable {
                       ^~~~~~~~~~~~~~~~~~~
Carthage/Checkouts/EVReflection/Source/EVReflection.swift:867:50: error: use of undeclared type 'EVCustomReflectable'
            let value: AnyObject = (theValue as! EVCustomReflectable).toCodableValue() as AnyObject
                                                 ^~~~~~~~~~~~~~~~~~~
Carthage/Checkouts/EVReflection/Source/EVReflection.swift:1249:34: error: use of undeclared type 'EVCustomReflectable'
                if !(original is EVCustomReflectable) {
                                 ^~~~~~~~~~~~~~~~~~~
```

This issue starts with the release version 5.5.0.
My Cartfile was:
  - github "evermeer/EVReflection" ~> 5.1.6.

For now, I had to modify the github command writing:
  - github "evermeer/EVReflection" == 5.4.5

**Comment by evermeer** [comment]
Indeed the file EVCustomReflectable.swift was missing from the Carthage target. I just added that and published that as version 5.5.3.

Thanks for reporting this!

**Comment by fedecastelli** [comment]
Thank you! It's working now!
